content/videos/challenges/18-3d-fractal-trees/index.json

```diff
@@ -8,6 +8,16 @@
   "topics": ["fractals", "generative art", "3D"],
   "canContribute": true,
   "timestamps": [],
-  "codeExamples": [],
+  "codeExamples": [
+    {
+        "title": "3D Fractal Trees",
+        "description": "This sketch visualizes 3D fractal trees in Processing.",
+        "image": "threeD.png",
+        "urls": {
+          "processing": "https://github.com/CodingTrain/Coding-Challenges/tree/main/018_SpaceColonizer3D/Processing",
+          "other": "https://github.com/CodingTrain/Coding-Challenges/tree/main/018_SpaceColonizer3D"
```

Comment:
Is there a reason why other is here as the same link as processing?

Reply:
I think it can be deleted.  See comment I left on spirograph.